mappings/net/minecraft/class_4576.mapping

```diff
@@ -1 +1,13 @@
 CLASS net/minecraft/class_4576
```

Comment:
Try to name this.

Reply:
Named BufferedBlockEntityRenderer not sure if this is "good" though

Reply:
`BatchedBlockEntityRenderer`?